tests/cross/src/test/scala/tests/pc/CompletionKeywordSuite.scala

```diff
@@ -186,50 +186,6 @@ class CompletionKeywordSuite extends BaseCompletionSuite {
        |""".stripMargin,
   )
 
-  check(
-    "match",
-    """
-      |package foo
-      |
-      |object A {
-      |  val x: Option[Int] = ???
-      |  val a = {
-      |    x ma@@
-      |  }
-      |}
-      |""".stripMargin,
-    """|match
-       |""".stripMargin,
-    filter = _ == "match",
-  )
-
-  checkEdit(
-    "match-edit".tag(IgnoreScala2),
-    """
-      |package foo
-      |
-      |object A {
-      |  val abc: Option[Int] = ???
-      |  val a = {
-      |    abc ma@@
-      |  }
-      |}
-      |""".stripMargin,
-    s"""
-       |package foo
-       |
-       |object A {
-       |  val abc: Option[Int] = ???
-       |  val a = {
-       |    abc match
-       |\tcase $$0
-       |
-       |  }
-       |}
-       |""".stripMargin,
-    filter = _ == "match",
-  )
```

Comment:
These tests were added with match case keyword completion in https://github.com/scalameta/metals/pull/4185/files
Now match isn't a keyword completion anymore, so thought we can remove these tests

Reply:
Should we keep it as a completion still though? Some people might now want to use the exhaustive check.

Reply:
For scala2 both exhaustive match and basic match aren't keyword completions, the same here. We can maybe keep basic match as a keyword completion and here have only the exhaustive one, but I wouldn't do it since it was already a special case in keyword completions